### DISCUSSION TITLE
HTML attribute 'rel' disappeared

### DISCUSSION THREAD

**Discussion by cllena-covers** [directus/directus#15905] [Archive]
Hi Directus Team,

For the work that I'm doing with Directus, I've made a big update leap from the alpha versions ([rc.82](https://github.com/directus/directus/releases/tag/v9.0.0-rc.82)) to [9.16.1](https://github.com/directus/directus/releases/tag/v9.16.1) and I've noticed that we can't add the 'rel' attribute using the 'Insert/Edit Link' tool in the WYSIWYG field as shown below.

![ef9e3c0d-9838-4c6d-9add-1f9afe14ca55](https://user-images.githubusercontent.com/90919009/194401972-492b89c8-f119-4a76-a15a-81838f3e70f7.png)

Here are some actions that I did in an attempt to get it working:
1. Updated to the newest version at the time of this writing 9.18.1
2. Overwrote `rel_list` as recommended in this discussion https://github.com/directus/directus/discussions/10387
3. Went to Settings/data-model and double checked that the 'Insert/Edit Link' is in the toolbar under the WYSIWYG field.

Before attempting any more effort in trying it to work.  I'd like to confirm, can we change the 'rel' attribute for any URL link in the WYSIWYG field in 9.16.1?

**Comment by azrikahar** [comment]
Technically it didn't "disappear", but the rc.82 screenshot is showing TinyMCE's default insert/edit link modal, whereas 9.16.1 screenshot shows Directus' custom add/edit link modal, as explained here: https://github.com/directus/directus/issues/12215#issuecomment-1070964834

With that said, it should still be possible to either use `Edit Source Code` in the editor, or `Edit Raw Value` in the field label to customize the `rel` and/or any other href attributes not readily available in the current modal. Here's a quick example:

https://user-images.githubusercontent.com/42867097/195044077-3550c0fe-f2d6-41fb-bd95-63f5ad29f646.mp4
